airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/util/ConnectorExceptionUtil.java

```diff
@@ -0,0 +1,55 @@
+package io.airbyte.integrations.util;
+
+import io.airbyte.commons.exceptions.ConfigErrorException;
+import io.airbyte.commons.exceptions.ConnectionErrorException;
+import io.airbyte.integrations.base.errors.messages.ErrorMessage;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.function.Predicate;
+
+public class ConnectorExceptionUtil {
+
+    private static final List<Predicate<Throwable>> configErrorPredicates = new ArrayList<>();
+    private static final String RECOVERY_CONNECTION_ERROR_MESSAGE = "We're having issues syncing from a Postgres replica that is configured as a hot standby server. " +
+            "Please see https://docs.airbyte.com/integrations/sources/postgres/#sync-data-from-postgres-hot-standby-server for options and workarounds";
+
+    static {
```

Comment:
nit : Can we just define this inline above? 
List<Predicate<Throwable>> CONFIG_ERROR_PREDICATES = List.of(pred1, pred2....)